packages/react-native/types/modules/globals.d.ts

```diff
@@ -187,6 +187,7 @@ declare interface Request extends Object, Body {
   readonly mode: RequestMode_;
   readonly referrer: string;
   readonly url: string;
+  readonly signal: AbortSignal | undefined;
```

Comment:
Shouldn't this be:

```suggestion
readonly signal: AbortSignal | null | undefined;
```

I think that this is how TS idiomatically describes nullable props.

Reply:
I'm torn with this 🤔 -- according to the fetch spec it should be initialised as null as it's optional:

> A [Request](https://fetch.spec.whatwg.org/#request) object has an associated signal (null or an [AbortSignal](https://dom.spec.whatwg.org/#abortsignal) object), initially null

_Source: https://fetch.spec.whatwg.org/#request-signal_

However, we look at the implementation in react native and it returns `undefined`:

```ts
new Request('https://facebook.com').signal
```

Reply:
Ok, thanks for digging more into this issue.